qgroundcontrol.pro

```diff
@@ -70,9 +70,21 @@ QGC_APP_COPYRIGHT   = "Copyright (C) 2019 QGroundControl Development Team. All r
 
 WindowsBuild {
     QGC_INSTALLER_SCRIPT        = "$$SOURCE_DIR\\deploy\\windows\\nullsoft_installer.nsi"
-    QGC_INSTALLER_ICON          = "$$SOURCE_DIR\\deploy\\windows\\WindowsQGC.ico"
-    QGC_INSTALLER_HEADER_BITMAP = "$$SOURCE_DIR\\deploy\\windows\\installheader.bmp"
     QGC_INSTALLER_DRIVER_MSI    = "$$SOURCE_DIR\\deploy\\windows\\driver.msi"
+
+    exists($$PWD/custom/deploy/windows/installheader.bmp) {
+        message("Using custom installheader.bmp")
+        QGC_INSTALLER_HEADER_BITMAP = "$$SOURCE_DIR\\custom\\deploy\\windows\\installheader.bmp"
+    } else {
+        QGC_INSTALLER_HEADER_BITMAP = "$$SOURCE_DIR\\deploy\\windows\\installheader.bmp"
+    }
+
+    exists($$PWD/custom/deploy/windows/WindowsQGC.ico) {
+        message("Using custom WindowsQGC.ico")
+        QGC_INSTALLER_ICON = "$$SOURCE_DIR\\custom\\deploy\\windows\\WindowsQGC.ico"
+    } else {
+        QGC_INSTALLER_ICON = "$$SOURCE_DIR\\deploy\\windows\\WindowsQGC.ico"
+    }
```

Comment:
Unless I'm missing something this is not needed. The way you override these sorts of things is by just setting them in custom.pri which is included after this. That will then in turn overwrite the values with the new ones. 

Otherwise you need these exists things all over the place on every single possible config setting.

Reply:
I haven't tested, but based on what you said this makes sense. For some reason I thought custom.pri was "evaluated" before this. I am dropping this one. Thank you!

Reply:
I did a quick test, overriding that from custom.pri, and for some reason is not working. I am trying to debug it now, but I thought I could post here the update.